freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -66,11 +71,74 @@ def handle(self, event):
             "Orignial nvrs of build in the advisory #{0} are: {1}".format(
                 db_event.search_key, " ".join(original_nvrs)))
 
-        msg = "Skipping due to being blocked on further implementation for now."
+        index_images = self._pyxis.get_operator_indices()
+        # get latest bundle images per channel per index image filtered by higher version
+        structured_bundles = self._pyxis.get_bundles_per_channel_per_index_image(index_images)
+        # get just bundles from structured data
+        all_bundles = []
+        for channel_bundle in structured_bundles.values():
+            for bundles in channel_bundle.values():
+                all_bundles.extend(bundles)
```

Comment:
I'm wondering if `get_bundles_per_channel_per_index_image` could just return the list of bundles you are creating here. This would simplify the code and reduce memory usage.

Reply:
Yes, I thought about it too. I will change it.